appendix-basic65.tex

```diff
@@ -672,17 +776,18 @@ \subsection{ASC}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$C6
 \item [Format:] {\bf ASC(}string{\bf)}
-\item [Usage:] Takes the first character of
-               the string argument and returns its numeric code value.
-               The name was apparently chosen to be a mnemonic to ASCII,
-               but the returned value is in fact the so-called PETSCII code.
+\item [Usage:] The PETSCII code of the first character of
```

Comment:
Ok, I noticed for almost all functions, you've removed any prefacing verb (like 'returns', 'generates', etc)

I kinda miss having a verb like 'Returns', it feels a bit empty without it.

So I'm wondering, for any functions, should we replace '[Usage]:' with '[Returns]:'

Thanks to your recent label refinements, hopefully we could easily catch all these cases by searching for "{BASIC 65 Functions!"

Reply:
Accepted the suggestion to change "Usage" to "Returns." Nice! Also found some commands that were mis-labeled as functions. :) Also I intentionally kept "Usage" in special cases where it was more important to use behavioral language even if it's technically a function, such as FN() and USR().

In my mind, "returns" is a C-like term and not a BASIC term, and would be a special term that needs to be defined as a synonym of "evaluates to." "Evaluates to" is more accurate at the level of abstraction for BASIC functions, operators, and expressions. But that's nit picking. :)